src/client/content_cao.cpp

```diff
@@ -473,8 +473,13 @@ void GenericCAO::setAttachment(int parent_id, const std::string &bone, v3f posit
 			o->removeAttachmentChild(m_id);
 		if (parent)
 			parent->addAttachmentChild(m_id);
+#if 0
+		printf("Attach id=%d to parent=%d (old=%d)\n", m_id, parent_id, old_parent);
+		std::cout << "\tpos=" << PP(position) << ", rot=" << PP(rotation) << std::endl;
```

Comment:
should probably be removed from here

Reply:
Every time I debug attachments I end up adding a similar snippet. I'd prefer to keep it here in case something breaks again.

Reply:
That's also fine but please change it to use `std::cout` then.